src/pubsub.js

```diff
@@ -78,6 +78,15 @@ class BasicPubSub extends Pubsub {
      * @returns {string} message id as string
      */
     this.defaultMsgIdFn = (msg) => utils.msgId(msg.from, msg.seqno)
+
+    /**
+     * Topic validator map
+     *
+     * Keyed by topic
+     * Topic validators are functions with the following input:
+     * (topic: string, peer: Peer, message: rpc.RPC)
```

Comment:
```suggestion
     * @type {Map<string, validator>}
```

You can define a typedef for the validator as below:

```
/**
 * Topic validator function.
 * @typedef {function(topic: string, peer: Peer, message: rpc.RPC): boolean} validator
 */
```